File: Estrutura de Dados II/LABS.UFSJ/LAB03/cubo.c/cubo.c
#include "cubo.h"

typedef struct
{
    double aresta;
}Cubo;

Cubo criarCubo()
{
    Cubo *c = (Cubo*)malloc(sizeof(Cubo));

    c->aresta = 0;

    return *c;
}

void setAresta(Cubo *c,double a)
{
    c->aresta = a;
}

//retorna lado 
double getLado(Cubo *c)
{
    return c->aresta;
}

//retorna area total do cubo 
double getArea(Cubo *c)
{
    return (c->aresta * c->aresta) * 6;
}

double getVolume(Cubo *c)
{
    return c->aresta * c->aresta * c->aresta;
}

File: Estrutura de Dados II/LABS.UFSJ/LAB03/cubo.java/cubo.java

public class cubo()
{

        private double aresta;

        //
        public cubo(double aresta) {this.aresta = 0;}

        //setters
        public void setAresta(double aresta) {this.aresta = aresta};


        //gettes
        public double getAresta() {return this.aresta;}
        public double getArea() {return (this.aresta * this.aresta) * 6;}
        public double getVolume() {return (this.aresta * this.aresta * this.aresta;)}
        
}